content/docs/command-reference/push.md

```diff
@@ -70,6 +70,8 @@ backward from the target [stage files](/doc/command-reference/run), through the
 corresponding [pipelines](/doc/command-reference/pipeline), to find data files
 to push.
 
+Note that as `dvc push` uploads tracked files and directories to remote storage, it won't upload files and directories listed under `.dvcignore`.<br> See [.dvcignore](docs/user-guide/.dvcignore) for more details.
```

Comment:
Are you sure this is the case? Even if the data was tracked before adding it to .dvcignore? If so this note is OK, just again: no `<br>` and needs formatting.